src/main/java/org/llorllale/cactoos/matchers/ScalarHasValue.java

```diff
@@ -82,9 +82,6 @@ protected boolean matchesSafely(final Scalar<T> item) {
     protected void describeMismatchSafely(final Scalar<T> item,
         final Description description) {
         description
-            .appendText("was ")
-            .appendValue(
-                new UncheckedScalar<>(item).value()
-            );
+            .appendValue(new UncheckedScalar<>(item).value());
```

Comment:
@borysfan why this change?

Reply:
Please take look at test case hasMismatchDescriptionForFailedTest (https://github.com/llorllale/cactoos-matchers/pull/97/files#diff-b1851e044751c294fbcb4a9467a5f931R86) Before that change on mismatch situation the description was: 'but was: was "something"'. @fabriciofx suggested that we should remove this duplication.

Reply:
@llorllale I've thought `was: was` strange... :)

Reply:
@borysfan @fabriciofx let's keep PRs tightly focused please! :)